plugins/source/pagerduty/docs/tables/pagerduty_services.md

```diff
@@ -0,0 +1,43 @@
+# Table: pagerduty_services
+
+
+
+The primary key for this table is **id**.
+
+## Relations
+
+The following tables depend on pagerduty_services:
+  - [pagerduty_service_rules](pagerduty_service_rules.md)
+
+## Columns
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id|UUID|
+|_cq_parent_id|UUID|
+|dependencies|JSON|
+|id (PK)|String|
+|type|String|
+|summary|String|
+|self|String|
+|html_url|String|
+|name|String|
+|description|String|
+|auto_resolve_timeout|Int|
+|acknowledgement_timeout|Int|
```

Comment:
Should these be ints? What are their underlying units?

Reply:
Ok, I see in the docs that it's in seconds. Don't think there's anything for us to do here